Makefile

```diff
@@ -307,6 +307,7 @@ endif
 # This will only work for maintainers.
```

Comment:
I am going to try to test if this works for locally running the drone CLI with a jsonnet file. Assuming not, we may want to add a drone-test or something that makes a .drone.yml file in the root of the project (probably .gitignore it) and doesn't sign it so people can run the drone exec command to test a pipeline locally. Then it is worth updating the .drone/README.md with the developer instructions on this path.

https://docs.drone.io/quickstart/cli/

Reply:
Confirmed in my testing the drone cli requires precisely a .drone.yml in the root of the project.

C:\workspace\go\src\agent>drone exec
2023/01/13 09:55:59 proto: duplicate proto type registered: PluginSpec
2023/01/13 09:55:59 proto: duplicate proto type registered: PluginPrivilege
2023/01/13 09:55:59 open .drone.yml: The system cannot find the file specified.

Reply:
I'll create a new Makefile target for `generate-drone-config`

Reply:
done, now users can generate the YAML separately with `make generate-drone` without needing to sign it.